zcash_extensions/Cargo.toml

```diff
@@ -11,7 +11,7 @@ rust-version = "1.60"
 
 [dependencies]
 blake2b_simd = "1"
-zcash_primitives = { version = "0.10", path = "../zcash_primitives", features = ["zfuture" ] }
+zcash_primitives = { version = "0.10", path = "../zcash_primitives", default-features = false, features = ["zfuture" ] }
```

Comment:
Here we don't need to bother, because this crate has never been published (and even if it had, a performance regression in Orchard wouldn't matter to this crate).